src/sentry/incidents/models/alert_rule.py

```diff
@@ -251,24 +274,31 @@ class AlertRule(Model):
     excluded_projects = models.ManyToManyField(
         "sentry.Project", related_name="alert_rule_exclusions", through=AlertRuleExcludedProjects
     )  # NOTE: This feature is not currently utilized.
-    name = models.TextField()
```

Comment:
I just moved things around so that `excluded_projects` and `include_all_projects` are next to each other